### PR TITLE
[SMALLFIX] Minor code improvement for package tachyon.worker.block.evictor

### PR DESCRIPTION
**Pull request by yupeng9**

### CODE REVIEW

File: servers/src/main/java/tachyon/worker/block/evictor/LRFUEvictor.java [hunk 1]
```diff
@@ -111,6 +111,9 @@ public LRFUEvictor(BlockMetadataManagerView view, Allocator allocator) {
    */
   private StorageDirView cascadingEvict(long bytesToBeAvailable, BlockStoreLocation location,
       EvictionPlan plan, List<Map.Entry<Long, Double>> sortedCRF) {
+    Preconditions.checkNotNull(location);
+    Preconditions.checkNotNull(plan);
+    Preconditions.checkNotNull(sortedCRF);
```
Comment (calvinjia) [inline servers/src/main/java/tachyon/worker/block/evictor/LRFUEvictor.java]: What's the convention for precondition checks? Should we still do it even if the values are isolated from user input?

Comment (yupeng9) [inline servers/src/main/java/tachyon/worker/block/evictor/LRFUEvictor.java]: Usually validation for user input is enough when CheckForNull is also used (http://findbugs.sourceforge.net/api/edu/umd/cs/findbugs/annotations/CheckForNull.html). With CFN compiler can provide such guarantee.

I can remove these checks if you feel it's too strong.

Comment (uronce-cc) [inline servers/src/main/java/tachyon/worker/block/evictor/LRFUEvictor.java]: +1 for calvin's comment. I felt it is not necessary for internal methods to check the input.

Comment (yupeng9) [inline servers/src/main/java/tachyon/worker/block/evictor/LRFUEvictor.java]: Sounds good to me. Done.

**Comment by calvinjia** [comment]
LGTM

**Pull request merged**